src/components/MultiSelect/MultiSelect.test.tsx

```diff
@@ -520,149 +457,142 @@ describe.skip("MultiSelect Dialog", () => {
 
   it("should check parent item if all child items are checked", () => {
     selectedTestItems = {
-      "multiselect-dialog-test-id": { items: ["red", "blue"] },
+      "multiselect-test-id": { items: ["red", "blue"] },
     };
 
     render(
       <MultiSelect
-        id="multiselect-dialog-test-id"
-        labelText="MultiSelect Label"
-        type="dialog"
+        id="multiselect-test-id"
+        helperText="multiselect-helper-text"
+        buttonText="multiselect-button-text"
+        defaultItemsVisible={defaultItemsVisible}
         items={items}
+        isSearchable={false}
+        isBlockElement={false}
         isDefaultOpen={true}
         selectedItems={selectedTestItems}
         onMixedStateChange={() => null}
         onChange={() => null}
         onClear={() => null}
-        onApply={() => null}
       />
     );
     expect(screen.getByLabelText("Colors")).toBeChecked();
   });
+
   it("should render a count button with the correct count, should clear the selectedItems on click ", () => {
-    const { container } = render(
-      <MultiSelectTestDialogComponent multiSelectId="multiselect-dialog-test-id" />
+    render(
+      <MultiSelectTestDialogComponent multiSelectId="multiselect-test-id" />
     );
-    // Check for the selectedItems count button to not be present
     expect(
-      container.querySelector("span[role='button']")
+      screen.queryByTestId("multi-select-close-button-testid")
     ).not.toBeInTheDocument();
+
     // Open menu
-    userEvent.click(screen.getByRole("button", { name: /MultiSelect Label/i }));
+    userEvent.click(screen.queryByRole("button"));
+
+    render(
```

Comment:
Use rerender here, too.

Reply:
Actually, you shouldn't need to render this component again here if you are clicking the button to open the component.